src/app/pages/ValidatorDetailsPage/BalanceDistributionCard.tsx

```diff
@@ -23,10 +23,6 @@ function chartData(t: TFunction, validator: Validator | undefined) {
   const otherShares = activeShares - selfDelegationShares
 
   return [
-    {
-      label: t('validator.active'),
-      value: activeShares,
-    },
```

Comment:
Was this intentional? Math is still `otherShares = activeShares - selfDelegationShares`

Reply:
it was intentional. It is used to compute otherShares, but showing active shares in chart does not make sense (as it is "entire pie chart"). side note: there is still a discussion about this section.